README.rst

```diff
@@ -1,3 +1,6 @@
+
+No longer synced.  `Actively maintained version <https://github.com/galaxy-genome-annotation/python-apollo>`__
```

Comment:
Hum, not sure what you mean here?

Reply:
That was from my old repo.  Should be gone now.